ingest/dns_to_s3.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
could you add a couple lines of comments detailing what this file does?
maybe just sentence...